docs/paper/paper.md

```diff
@@ -0,0 +1,85 @@
+---
+title: "VirtualShip for simulating oceanographic fieldwork anywhere in the global ocean"
+tags:
+  - Python
+  - oceanography
+  - fieldwork simulation
+  - Lagrangian modelling
+  - data ingestion
+authors:
+  - name: Jamie R. C. Atkins
+    orcid: 0000-0002-5735-3312
+    corresponding: true
+    affiliation: 1
+  - name: Emma Daniels
+    orcid: 0009-0005-9805-5257
+    affiliation: 1
+  - name: Nick Hodgskin
+    affiliation: 1
+  - name: Aart Stuurman
+  - name: Iury Simoes-Sousa
+    orcid: 0000-0002-2484-510X
+    affiliation: 2
+  - given-name: Erik
+    dropping-particle: van
+    surname: Sebille
+    orcid: 0000-0003-2041-0704
+    affiliation: 1
+
+affiliations:
+  - name: Institute for Marine and Atmospheric Research, Utrecht University, the Netherlands
+    index: 1
+  - name: Woods Hole Oceanographic Institution, Falmouth, MA, USA
+    index: 2
+date: 4 December 2025
+bibliography: paper.bib
+---
+
+# Summary
+
+`VirtualShip` is a Python-based package which exploits the customisability of the open-source `Parcels` Lagrangian simulation framework [@Lange2017; @Delandmeter2019] to simulate measurements as if they were coming from real-life oceanographic instruments. The software builds a virtual ocean world by streaming data from the [Copernicus Marine Data Store](https://marine.copernicus.eu/) on-the-fly, facilitating virtual expeditions anywhere on the globe.
```

Comment:
Maybe mention that the tool can be used for training students, cruise planning as well as design of instrumentation and sampling strategies.